bokehjs/test/models/axes/axis.coffee

```diff
@@ -32,6 +34,43 @@ describe "Axis", ->
     })
     expect(axis.compute_labels([0,2,4.0,6,8,10])).to.be.deep.equal ["zero", "2", "four", "6", "8", "ten"]
 
+  it "loc should return numeric fixed_location", ->
+    doc = new Document()
+    p = new Plot({
+      x_range: new Range1d({start: 0, end: 10})
+      y_range: new Range1d({start: 0, end: 10})
+    })
+    doc.add_root(p)
+    ticker = new BasicTicker()
+    formatter = new BasicTickFormatter()
+    axis = new Axis({
```

Comment:
You are constructing an abstract class here. This is btw. why I want to have all new tests in typescript, to avoid such mistakes.

Reply:
I am not opposed, I can just only handle so much at a time. In any case given how much implementation is inside Axis I think it needs to be concrete, just so we can test it. Are you proposing any specific change here right now or is this good to merge?